src/client/app/chat/chat.component.css

```diff
@@ -458,6 +458,10 @@ textarea:focus {
     cursor: pointer;
 }
 
+.dropdown-menu {
```

Comment:
What's this for?

Reply:
For attachment button size on chat page. After removing the appear component uneccessary space was showing so I added this.

Reply:
Got it 👍